app/src/main/java/org/fossasia/pslab/activity/ControlActivity.java

```diff
@@ -20,8 +29,35 @@
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        //setContentView();
+        setContentView(R.layout.activity_control);
         scienceLab = ScienceLabCommon.getInstance().scienceLab;
-        ButterKnife.bind(this);
+        BottomNavigationView bottomNavigationView = (BottomNavigationView) findViewById(R.id.navigation);
+
+        bottomNavigationView.setOnNavigationItemSelectedListener
+                (new BottomNavigationView.OnNavigationItemSelectedListener() {
+                    @Override
+                    public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                        Fragment selectedFragment = null;
+                        switch (item.getItemId()) {
+                            case R.id.action_item1:
+                                selectedFragment = ControlFragmentMain.newInstance();
+                                break;
+                            case R.id.action_item2:
+                                selectedFragment = ControlFragmentRead.newInstance();
+                                break;
+                            case R.id.action_item3:
+                                selectedFragment = ControlFragmentAdvanced.newInstance();
+                                break;
+                        }
+                        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
+                        transaction.replace(R.id.frame_layout, selectedFragment);
```

Comment:
I wrote quite some part before writing the naming convention, so this mistake occurred. I would correct the same and other things pointed out in another PR which I would create today. I have already modified my local repo a lot, so if add another commit now a lot of extra things will be added. Rest assured this will be corrected.